src/components/VoteButton.jsx

```diff
@@ -0,0 +1,24 @@
+import { useState } from "react";
+
+function VoteButton() {
+  const [vote, setVote] = useState(0);
+  return (
+    <div>
+      <button
+        className="green-button"
+        onClick={() => setVote((currentVote) => currentVote + 1)}
```

Comment:
At the moment, we are optimistically rendering the votes which is great. However, upon refreshing the votes reset. We need to patch the votes in the database as well as optimistically rendering.